deploy/hawtio-embedded/src/main/java/io/hawt/embedded/Main.java

```diff
@@ -17,6 +17,14 @@
  */
 package io.hawt.embedded;
 
+import org.eclipse.jetty.ee10.webapp.WebAppContext;
+import org.eclipse.jetty.server.*;
+import org.eclipse.jetty.server.handler.ContextHandlerCollection;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
```

Comment:
Nitpicky but we prefer to place `java.*` and `javax.*` earlier than the rest. If you haven't changed the preferences of imports in your IDE, please change it for the hawtio project and fix the ordering.

Reply:
For example, in IDEA it should look like this:

![image](https://github.com/hawtio/hawtio/assets/156692/568e9d0a-76c0-4caa-827f-bcca2e00289e)

In doing so, we can put `jakarta.*` between `javax.*` and all the other imports.